SocketIOClientSwift/SocketIOClientOption.swift

```diff
@@ -68,6 +69,8 @@ public enum SocketIOClientOption: ClientOption {
             return .ReconnectAttempts(value as! Int)
         case "reconnectWait" where value is Int:
             return .ReconnectWait(value as! Int)
+        case "forceNew" where value is Bool:
```

Comment:
Also this option really doesn't make sense anymore. Maybe I'm not understanding how socket.io-client is supposed to handle reconnects (the protocol is kinda sketchy,) but it seems that it should always get a new sid on reconnect. So this option would only be good if there's some bug in the engine after a reconnect attempt, and you want to have a new engine every time.